week19/restock-shopping-cart_starter/cart.jsx

```diff
@@ -53,47 +44,16 @@ const useDataApi = (initialUrl, initialData) => {
   }, [url]);
 
 
-  return [state, setUrl];
-};
-
-const dataFetchReducer = (state, action) => {
-  switch (action.type) {
-    case "FETCH_INIT":
-      return {
-        ...state,
-        isLoading: true,
-        isError: false,
-      };
-    case "FETCH_SUCCESS":
-      return {
-        ...state,
-        isLoading: false,
-        isError: false,
-        data: action.payload,
-      };
-    case "FETCH_FAILURE":
-      return {
-        ...state,
-        isLoading: false,
-        isError: true,
-      };
-    default:
-      throw new Error();
-  }
+  return [inventory, setUrl];
```

Comment:
useDataApi returns the inventory state variable and the function to set the url state variable.